SurfaceTopography/Uniform/Autocorrelation.py

```diff
@@ -122,13 +124,19 @@ def autocorrelation_from_profile(topography, direction=0):
 
         r = sx * np.arange(nx) / nx
 
+    # The factor of two comes from the fact that the short cutoff is estimated
+    # from the curvature but the ACF is the slope, see arXiv:2106.16103
+    short_cutoff = topography.short_reliability_cutoff() if reliable else 0
```

Comment:
According to the line below, shouldn't it be None instead of 0 ?

Reply:
Good catch, fixed this.